### PR TITLE
#15/UI/BlueLabel

### PR DESCRIPTION
**Pull request by KatarzynaChmielecka**
image

### CODE REVIEW

File: src/Atoms/GreyLabel/GreyLabel.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const GreyLabel = (props) => {
+  return (
+    <GreyLabelContainer>
+      <p className="greyLabelText">{props.text}</p>
+    </GreyLabelContainer>
+  );
+};
+
+GreyLabel.propTypes = {
+  text: PropTypes.string,
```
Comment (MonikaKrella) [inline src/Atoms/GreyLabel/GreyLabel.js:13]: shouldn't be text as required prop?

**Review by MonikaKrella — CHANGES_REQUESTED** [review]

File: src/Atoms/GreyLabel/GreyLabel.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+import GreyLabelContainer from './GreyLabelStyled';
+
+const GreyLabel = (props) => {
```
[the remark this answers is not in the document]
Comment (Smyku6) [reply src/Atoms/GreyLabel/GreyLabel.js:4]: exactly `const GreyLabel = ( { text } ) => {`

File: src/Atoms/GreyLabel/GreyLabel.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  return (
+    <GreyLabelContainer>
+      <p className="greyLabelText">{props.text}</p>
```
Comment (Smyku6) [inline src/Atoms/GreyLabel/GreyLabel.js:7]: so if You destructure You can use here `{text}`

File: src/Atoms/GreyLabel/GreyLabelStyled.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import FONTS from '../../Styles/fontsStyledComponents';
+import COLORS from '../../Styles/varsStyledComponents';
+
+const GreyLabelContainer = styled.div`
```
Comment (Smyku6) [inline src/Atoms/GreyLabel/GreyLabelStyled.js:5]: change name of this Component to the same as filename - `GreyLabelStyled`

File: src/Atoms/GreyLabel/GreyLabelStyled.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  background: ${COLORS.grey00};
+  height: 2.5rem;
+  font-family: Mulish;
```
Comment (Smyku6) [inline src/Atoms/GreyLabel/GreyLabelStyled.js:8]: You can erase this line - we had this in our index.css at this moment 
`* {
  font-family: 'Mulish', sans-serif
}`

**Review by Smyku6 — CHANGES_REQUESTED** [review]

**Review by Smyku6 — APPROVED** [review]
Good GreyLabel Kasia!
image

**Pull request merged**